finetune/util/metrics.py

```diff
@@ -185,7 +247,27 @@ def sequences_overlap(true_seq, pred_seq):
     return start_contained or end_contained
 
 
-def sequence_labeling_overlaps(true, predicted):
+def sequence_exact_match(true_seq, pred_seq):
+    """
+    Boolean return value indicates whether or not seqs are exact match
+    """
+    return (
+            pred_seq["start"] == true_seq["start"]
+            and pred_seq["end"] == true_seq["end"]
+        )
+
+
+def sequence_superset(true_seq, pred_seq):
+    """
+    Boolean return value indicates whether or predicted seq is a superset of target
+    """
+    return (
```

Comment:
Slightly funky formatting for some reason

Reply:
fixed.